google-cloud-language/test/google/cloud/language/default_encoding_test.rb

```diff
@@ -0,0 +1,149 @@
+# Copyright 2017 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "helper"
+
+describe Google::Cloud::Language, :default_encoding, :mock_language do
+  it "detects the default encoding" do
```

Comment:
The CI servers are configured to use the Ruby defaults, so I am not worried about this failing. I left this test un-stubbed because if it ever does fail I would like to know.